pegasus/sites.v3/hourofcode.com/i18n/countries.json

```diff
@@ -532,7 +532,7 @@
   "uk": {
     "full_name": "United Kingdom",
     "default_language": "gb",
-    "solr_country_code": "gb",
+    "country_code": "gb",
```

Comment:
I'm not sure if this still works or if we need this at all. This was because United Kingdom was sometimes codes as `gb` and other times as `uk`.

Reply:
I think it's still needed so that we retrieve the hoc country count correct.  See https://github.com/code-dot-org/code-dot-org/pull/24667/files#diff-e718f776bb6d6d528ea9304a75a4898dR151